guides/getting-started/index.md

```diff
@@ -109,22 +113,38 @@ If you need to stop the Docker containers, you can use `yarn stop-prereqs`.
 These commands will vary across applications, so check out the README
 if you are looking at different Cardstack projects.
 
-### Start the front end application
+## Start the front end application
 
 Next, let's start the front end and see Cardstack in action.
 
 ```
 yarn start-ember
 ```
 
-Visit [http://localhost:4200](http://localhost:4200) to see the main Cardboard app in action:
-
-![screenshot of the Cardboard project](/images/cardboard-initial.png)
+Visit [http://localhost:4200](http://localhost:4200) to see the main app in action:
 
 Your newly added Card can be found at [http://localhost:4200/photographers/1](http://localhost:4200/photographers/1).
 
-## Next steps
+## Preview editing mode with the Edges
+
+The Card SDK provides an editing mode for Cards, called Edges.
+To see what it looks like, add the following to `cardhost/app/templates/application.hbs`:
+
+```handlebars
+{{#mock-login as |login|}} <button {{action login}}>View editor</button>{{/mock-login}}
+```
+
+Restart your local servers with `Control-c`, `yarn start-hub`, and `yarn start-ember`.
+
+Now when you visit [http://localhost:4200/photographers/1](http://localhost:4200/photographers/1), you can click "edit content", select the arrow icon in the top right corner. You will see the Right Edge, which is automatically generated from the Card Schema!
+You won't be able to make any edits yet, because we did not set up permissions, so keep reading these Guides!
```

Comment:
Maybe we should use 'grants' instead of 'permissions' just to be consistent with the syntax